src/main/java/com/zendesk/maxwell/MaxwellConfig.java

```diff
@@ -368,7 +372,10 @@ private void setup(OptionSet options, Properties properties) {
 		if ( properties != null ) {
 			for (Enumeration<Object> e = properties.keys(); e.hasMoreElements(); ) {
 				String k = (String) e.nextElement();
-				if (k.startsWith("kafka.")) {
+				if (k.startsWith("custom.")) {
+				    this.customProperties.setProperty(k.replace("custom.", ""), properties.getProperty(k));
+				}
+				else if (k.startsWith("kafka.")) {
```

Comment:
stlyistic nit here and other places: `else` on the same line as braces.

Reply:
On it.